modules/nf-core/yte/tests/main.nf.test

```diff
@@ -0,0 +1,57 @@
+nextflow_process {
+
+    name "Test Process YTE"
+    script "../main.nf"
+    process "YTE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "yte"
+
+    test("yaml - no_template") {
```

Comment:
This name is confusing to me, as it seems that you are passing a template...

Reply:
The file that is usally used as input is a template yaml. I have not created one yet so I am just passing a simple yaml file. (The file ending stays the same) I wanted to make that visible in the test name.

Reply:
Can I suggest a comment to that effect then.